components/list/list.js

```diff
@@ -37,6 +37,9 @@ class List extends LitElement {
 			:host {
 				display: block;
 			}
+			:host([hidden]) {
+				display: none;
+			}
```

Comment:
I've changed the demo to just use list directly (removing 2 demo files), but list wasn't handling `hidden` properly so this fixes that.